### PR TITLE
[Fix] LoadingGif() while loading unfollowers.

### LINKED ISSUE #1256: Show loading GIF when fetching "Unfollowers"
Right now, when fetching the user's unfollowers, the screen just waits and does nothing until the list is loaded, and then it is displayed. It'd be nice to instead show the user that the list is being loaded with the loading GIF

### PR DESCRIPTION
**Pull request by vineetvk01**
**Task:**
Show Loading while fetching unfollowers from instagram API.

**Preview:**
loading-fix

Closes #1256

### CODE REVIEW

File: browser/js/renderers.js [hunk 1]
```diff
@@ -388,23 +388,31 @@ function renderMessageSeenText (container, chat_) {
...
-      li.remove();
-    };
-    li.appendChild(unfollowButton);
-    ul.appendChild(li);
-  });
-
-  showInViewer(ul);
+  if (typeof users === 'string' && users === 'loading') {  
+    ul.appendChild(getLoadingGif());
+    showInViewer(ul);
```
Comment (ifedapoolarewaju) [inline browser/js/renderers.js]: I think we just `return` early here, and drop the `else {..}` block. The tree is less nested that way

File: browser/js/renderers.js [hunk 2]
```diff
@@ -388,23 +388,31 @@ function renderMessageSeenText (container, chat_) {
...
+        li.remove();
+      };
+      li.appendChild(unfollowButton);
+      ul.appendChild(li);
+    });
+    const unfollowers = document.querySelector('.unfollowers');
+    if (unfollowers) {
+      unfollowers.innerHTML = '';
+      unfollowers.appendChild(ul);
+    }
```
Comment (ifedapoolarewaju) [inline browser/js/renderers.js]: I think theres no need for this extra check. We can just call the `showInViewer` function and let it handle that. Also it seems from this current code, you are appending a `ul` into a `ul` which I think is not desired.

File: browser/js/renderers.js (hunk 1 above)
Comment (vineetvk01) [reply browser/js/renderers.js]: Agree @ifedapoolarewaju . :dagger:

File: browser/js/renderers.js (hunk 2 above)
Comment (vineetvk01) [reply browser/js/renderers.js]: Yes, i updated it with a **div** container to hold the **ul**

File: browser/js/renderers.js [hunk 3]
```diff
@@ -403,8 +409,11 @@ function renderUnfollowers (users) {
     li.appendChild(unfollowButton);
     ul.appendChild(li);
   });
-
-  showInViewer(ul);
+  const unfollowers = document.querySelector('div#unfollowers-view');
+  if (unfollowers) {
+    unfollowers.innerHTML = '';
+    unfollowers.appendChild(ul);
+  }
```
Comment (ifedapoolarewaju) [inline browser/js/renderers.js]: what is the need to do it this way if we can just call `showInViewer(ul);`? Is there something I'm missing somewhere? 🤔

Comment (vineetvk01) [reply browser/js/renderers.js]: If the viewer is currently having loadingGif() and the user closes the viewer (x) _**before the data loads**_, The list of un-followers will still pop up when the front-end will receive data. But here if the "div#unfollower-view" is absent then list wont render.

Comment (ifedapoolarewaju) [reply browser/js/renderers.js]: right, makes sense. Thank you for clarifying 👍

**Pull request merged**